docs/developer-docs/latest/developer-resources/database-apis-reference/rest-api.md

```diff
@@ -0,0 +1,642 @@
+`GET /api/restaurants?filters[id][$in][0]=3&filters[id][$in][1]=6&filters[id][$in][2]=8`
+:::
+
+::::tip
+Strapi takes advantage of the capability of [`qs`](https://github.com/ljharb/qs) to parse nested objects to create more complex queries.
+Use `qs` directly to generate complex queries instead of creating them manually.
+
+:::details Example
+
+<!-- ? devs, could you please check that the generated URL and object syntax is correct? -->
+```js
+const query = qs.stringify({
+  filters: {
+    $or: [
+      {
+        date: {
+          $eq: '2020-01-01'
+        },
+      },
+      {
+        date: {
+          $eq: '2020-01-02'
+        },
+      }
+    ],
+    title: {
+      $eq: 'hello'
+    },
+    author: {
+      name: {
+        $eq: 'Kai doe'
+      },
+    },
+  },
+});
+
+await request(`/api/books?${query}`);
+// GET /api/books?filters[$or][0][date][$eq]=2020-01-01&filters[$or][1][date][$eq]=2020-01-02&filters[title][$eq]=hello&filters[author][name][$eq]=Kai%20doe
+```
+
+:::
+::::
+
+#### Deep filtering
+
+:::request Example request: Find restaurants owned by a chef who belongs to a restaurant with star equal to 5
+<!-- ? how should I update this request example (filters syntax)? -->
+`GET /api/restaurants?filters[chef.restaurant.star][$eq]=5`
+:::
+
+::: caution
+
+- Querying your API with deep filters may cause performance issues.  If one of your deep filtering queries is too slow, we recommend building a custom route with an optimized version of your query.
+- Deep filtering isn't available for polymorphic relations. This relation type is used in `media`, `component` and `dynamic zone` fields.
+
+:::
+
+### Sort
+
+Queries can accept a `sort` parameter with the following syntax:
+
+`GET /api/:pluralApiId?sort=field1,field2`
+
+The sorting order can be defined with `:asc` (ascending order, default, can be omitted) or `:desc` (for descending order).
+
+:::request Example requests: Sort users by email
+`GET /api/users?sort=email` to sort by ascending order (default)
+
+`GET /api/users?sort=email:desc` to sort by descending order
+:::
+
+<!-- ? why do we specify `:asc`? if we write `title,price:desc`, is it equivalent to  `title:asc,price:desc`? -->
+:::request Example requests: Sort books on multiple fields
+`GET /api/books?sort=title,price:asc`
+
+`GET /api/books?sort=title,author.name`
+
+`GET /api/books?sort[0]=title&sort[1][author]=name` using an array format
+
+:::
+
+### Pagination
+
+Queries can accept `pagination` parameters. Results can be paginated either by page or by offset.
+
+:::note
+Pagination methods can not be mixed. Always use either `page` with `pageSize` **or** `start` with `limit`.
+:::
+
+#### Pagination by page
+
+Use the following parameters:
+
+<!-- ? is it 100 or 10 for default page size -->
+
+| Parameter              | Description | Default |
+| ---------------------- | ----------- | ------- |
+| `pagination[page]`     | Page number | 1       |
+| `pagination[pageSize]` | Page size   | 100     |
```

Comment:
you can add `pagination[withCount]` Boolean type => (0,false,f / 1,true,t)

Triggers the return of the total / total + pageCount fields.

Reply:
And default is to true